modules/subchannel/doc/content/source/scmclosures/SCMHTCBorishanskii.md

```diff
@@ -0,0 +1,49 @@
+# SCMHTCBorishanskii
+
+!syntax description /SCMClosures/SCMHTCBorishanskii
+
+The HTC closure models inherit from: [SCMHTCClosureBase](SCMHTCClosureBase.md).
+
+## Borishanskii Correlation for Turbulent Nusselt Number
+
+!! Intentional comment to provide extra spacing
+
+The Borishanskii correlation [!cite](borishanskii1969heat) is used for calculating the Nusselt number in fuel-pin bundles, considering the geometry of the bundle.
```

Comment:
looking at the paper this was obtained for a triangular array with 7 pins. It s unlikely to be valid for quadrilateral arrays